src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    private boolean isDone;
+    private final String taskName;
+
+    public Task(String taskName) {
+        this.isDone = false;
+        this.taskName = taskName;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void setDone(boolean done) {
```

Comment:
done does not convey the boolean nature of the variable that isDone conveys. perhaps it would be better to do:
public void setDone(boolean isDone) {
    this.isDone = isDone;
}